### DISCUSSION TITLE
Large do input (Variante)

### DISCUSSION THREAD

**Discussion by pihcardozo** [Telefonica/mistica-design#1344] [Proposals]
### Goal

Entender a possibilidade de termos uma versão large do input e qual impacto isso teria no DS global.

### Context

Precisamos de uma versão large do input para ser aplicado com destaque no layout ao ser utilizado sozinho.

### Impact

Teríamos uma versão maior do input, o que contribuiria para uma variedade de possibilidades de aplicação. 

### Discussion Questions

A versão large se aplica para todas as variantes existentes do input atual?

Como ficaria a variante “Read only” já que o input não tem bordas? 

### Do you have a visual proposal?

https://www.figma.com/file/o4buhch2d9GQhU5txrTNmO/Large-do-input-(Variante)?type=design&node-id=110%3A14245&mode=design&t=4njhXSAd11ESnV59-1

**Comment by AnaMontes11** [comment]
Hey @pihcardozo! Thanks for reaching out.

Reviewing the information you've provided, we understand that this need for input large goes beyond functionality and is solely related to an aesthetic aspect. In that case, we believe that the current Mistica input works.

We have a few questions though. For instance, what happens when we have more inputs? Will some be in large size while others remain as they are now? What about longer texts?

Could you show us more use cases? Examples from different contexts and with other typologies (not just numeric). This would help us have a broader understanding to address it.

Thanks!

**Reply by vinrosa** [reply]
Hello @AnaMontes11 

We wouldn't have more than one input on the screen, this component variation was created precisely for flows where the user's focus is on filling out a single input. In situations where the screen has more than one element, be it another input or any other element, a regular input must be used.

The intention is that the user, when entering this screen, has the focus on the input with the keyboard already active for filling.

Use cases where input large would be used
![image](https://github.com/Telefonica/mistica-design/assets/129896641/2a3289b8-6ac4-409a-8cb9-24d85830aa09)

Examples where normal input should be used
![image](https://github.com/Telefonica/mistica-design/assets/129896641/1e766df5-31d5-43ba-a57b-e67122d1ecf7)

**Reply by AnaMontes11** [reply]
Hi @vinrosa, for the purpose you mentioned, the `<autofocus>` property already exists, which opens the keyboard. So, using the input we already have shouldn't be an impediment.

**Comment by yceballost** [comment]
This component won't be part of Mística by the moment, will be developed in a Vivo extended library
